pkg/types/aws_resource_manager.go

```diff
@@ -83,6 +83,8 @@ type AWSResourceManager interface {
 	// This method also adds/updates the ConditionTypeReferencesResolved for the
 	// AWSResource.
 	ResolveReferences(context.Context, client.Reader, AWSResource) (AWSResource, error)
+	// IsSynced returns true if a resource is synced.
```

Comment:
I thought you were going to add this method in `AWSResourceManager` as per following comment?
https://github.com/aws-controllers-k8s/code-generator/pull/247#discussion_r760544423